### PR TITLE
THOR 517 Data table column reorderable

### PR DESCRIPTION
**Pull request by Konnoke**

### CODE REVIEW

**Review by rartiss55 — DISMISSED** [review]

File: src/app/components/data-table/data-table.component.ts [hunk 1]
```diff
@@ -74,6 +75,7 @@ export class DataTableOptions extends BaseNeonOptions {
         this.filterable = this.injector.get('filterable', false);
         this.heatmapDivisor = this.injector.get('heatmapDivisor', 0);
         this.ignoreSelf = this.injector.get('ignoreSelf', false);
+        this.reorderable = this.injector.get('reorderable', true);
```
Comment (ThomasSchellenbergNextCentury) [inline src/app/components/data-table/data-table.component.ts:78]: Please add reorderable to subGetBindings.

File: src/app/components/data-table/data-table.component.html [hunk 2]
```diff
@@ -200,7 +200,7 @@
                 [rowHeight]="getTableRowHeight()"
                 [scrollbarH]="true"
                 [scrollbarV]="true"
-                [reorderable]="false"
+                [reorderable]="options.reorderable"
```
Comment (ThomasSchellenbergNextCentury) [inline src/app/components/data-table/data-table.component.html:210]: Please add a mat-button-toggle-group for reorderable to toggle true or false.

**Review by ThomasSchellenbergNextCentury — CHANGES_REQUESTED** [review]

File: src/app/components/data-table/data-table.component.ts (hunk 1 above)
Comment (Konnoke) [reply src/app/components/data-table/data-table.component.ts:78]: Added in the latest commit

File: src/app/components/data-table/data-table.component.html (hunk 2 above)
Comment (Konnoke) [reply src/app/components/data-table/data-table.component.html:210]: Added in the latest commit

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]

**Pull request merged**